questions.ts

```diff
@@ -303,18 +303,21 @@ export const askQuestions = async (
       COMMUNITY_OWNER,
       COMMUNITY_REPO,
     );
-    const { communityProjectPath } = await prompts(
+    const { communityProjectConfig } = await prompts(
       {
         type: "select",
-        name: "communityProjectPath",
+        name: "communityProjectConfig",
         message: "Select community template",
-        choices: projectOptions,
+        choices: projectOptions.map(({ title, value }) => ({
+          title,
+          value: JSON.stringify(value), // serialize value to string in terminal
+        })),
         initial: 0,
       },
       handlers,
     );
-    program.communityProjectPath = communityProjectPath;
-    preferences.communityProjectPath = communityProjectPath;
+    program.communityProjectConfig = communityProjectConfig;
+    preferences.communityProjectConfig = communityProjectConfig;
```

Comment:
you can parse the JSON here (should work as for datasource we're already using a JSON type) and then do `communityProjectConfig?: CommunityProjectConfig` in `types.ts`
```suggestion
    program.communityProjectConfig = JSON.parse(communityProjectConfig);
    preferences.communityProjectConfig = JSON.parse(communityProjectConfig);
```